### ISSUE TITLE
Error importing any file 

### ISSUE THREAD

**Issue by glezmoreno** [IDEMSInternational/R-Instat#7357]
I'm trying to test R-instant but the import data tool does not work as it indicates "cannot import this file with the current options". I first tried with my own CSV using separated comma, semicolon, deleting columns and changing the options but it didn't work. Finally I tested the tutorial and tried to add the dodoma data from the climatic dataset and it can't be imported either

![image](https://user-images.githubusercontent.com/73281402/161809141-f9daaf44-9f84-4d8d-823e-cdb225db128a.png)

I have installed the last version from http://r-instat.org/Download.html for both 32 and 64 bits. Thanks!!

**Comment by shadrackkibet** [comment]
@glezmoreno thanks for reporting the following bug. 

Not sure what's happening from your end. This might be caused by the previous steps. If you could provide more information on what you did before trying to load the dataset then I can try to replicate it. 

In the meantime, I suggest you try the following.

1) Close R-Instat using the close button(Red X  - Top left).
![image](https://user-images.githubusercontent.com/26301973/161917215-d0f43eee-0c25-4faa-ae21-e36f3414e4c1.png)
2) Start R-Instat again.
3) Try loading the dataset again by navigating to **File>Import From Library>Load From Instat Collection>Browse>Climatic>Climatic_guide_dataset.xlsx>Open**
Select dodoma as shown below and then click OK;
![image](https://user-images.githubusercontent.com/26301973/161917969-91cf01b2-7576-4017-ba3e-aec81895ebff.png)

Let us know if this works for you.

**Comment by glezmoreno** [comment]
Yes, I have tried to close the software and reboot but it still shows the same problem. The steps were basic: install R-Instat and try to load dataset using the same route as you told me File>Import From Library>Load From Instat Collection>Browse>Climatic>Climatic_guide_dataset.xlsx>Open

In fact the problem seems to be general with any dataset. If I try to open any dataset from R library it gives also an error (see below).  I did the following steps: File>Import From Library>Load From R> Air quality>Ok

I'm running a Windows  10 Home  64 bits and I used daily R and R Studio. I have several versions of R installed. I have even tried to install the last version of R 4.1.3 and still gives the same error. Is there any way I can see an error log or diagnostics of the software so I can share with you? My idea was to use R-instant for teaching basic stats but I'm not sure if my students will have similar problem. 

![image](https://user-images.githubusercontent.com/73281402/162017461-b527bae5-4a8a-4dd7-9127-9d6acd73e894.png)

**Comment by shadrackkibet** [comment]
Happy to take this discussion to an email/skype/zoom and discuss it further if you are happy?

This looks like a different error message. You may try deleting the following file `etc/Renviron.site` and see if it works. Refer to the solution provided in https://github.com/rdotnet/rdotnet/issues/62#issuecomment-369065108

**Comment by dannyparsons** [comment]
Unless this file is still on the machine somewhere from a previous R installation?

**Comment by dannyparsons** [comment]
Here's a further suggested solution in this thread https://community.rstudio.com/t/unable-to-load-shared-object-on-startup-in-rstudio-only-r4-0-0-rstudio-1-2-5042/67069/3
